src/test/components/ProfileCallTreeView.test.js

```diff
@@ -8,17 +8,22 @@ import ProfileCallTreeView from '../../components/calltree/ProfileCallTreeView';
 import renderer from 'react-test-renderer';
 import { Provider } from 'react-redux';
 import { storeWithProfile } from '../fixtures/stores';
-import {
-  getProfileForInvertedCallTree,
-  getProfileForUnfilteredCallTree,
-} from '../fixtures/profiles/profiles-for-call-trees';
+import getProfileFromTextSamples from '../fixtures/profiles/text-to-profile';
 import { changeCallTreeSearchString } from '../../actions/profile-view';
 import { getBoundingBox } from '../fixtures/utils';
 
 describe('calltree/ProfileCallTreeView', function() {
+  const { profile } = getProfileFromTextSamples(`
+    A A A
+    B B B
+    C C H
+    D F I
+    E G
+  `);
```

Comment:
There is a significant change: you use the same profile everywhere, while before we were returning a new one for every test, which is better for unit tests. I don't _think_ this is a problem at the moment as I think we replaced all code that mutates the profile, but I think this could still lead to surprises later.

Do you think getProfileFromTextSamples could return a function that would return a new profile at each call ?

Reply:
But maybe this is something we can do later, as a lot of other tests are doing the same _already_.